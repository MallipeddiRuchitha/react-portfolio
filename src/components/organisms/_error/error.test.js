import React from 'react';

import * as Constants from "../../../constants";
import Error from './_error';
import { shallow ,mount} from 'enzyme';
describe("Error component",() =>{
    test("Matches the snapshot",() => {
        const error = shallow(<Error />);
        expect(error).toMatchSnapshot();

    })
    test("Error element is rendered",() => {
        const error = shallow(<Error />);
        expect(error.find("#error").exists()).toEqual(true);
        
    });
    test("is Link element present ",() => {   
        const error = shallow(<Error />);
        expect(error.find("#link").prop('href')).toBe("/");});

});