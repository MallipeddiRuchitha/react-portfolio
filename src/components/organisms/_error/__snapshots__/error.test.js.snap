// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component Matches the snapshot 1`] = `
<div
  className="makeStyles-styleError-3"
  id="error"
>
  <WithStyles(ForwardRef(Typography))
    className="makeStyles-inputError-1"
    variant="h3"
  >
    You have to login first
  </WithStyles(ForwardRef(Typography))>
  <a
    className="makeStyles-link-2"
    href="/"
    id="link"
  >
     
    Back to login page
  </a>
</div>
`;
